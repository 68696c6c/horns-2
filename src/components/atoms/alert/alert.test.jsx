/* global describe, it, expect */
import React from 'react'

import { render } from 'setup-test'

import Alert from '.'

describe('Alert', () => {
  it('should render as default', () => {
    const { container } = render(<Alert>example</Alert>)
    expect(container).toMatchSnapshot()
  })
  it('should render as block', () => {
    const { container } = render(<Alert block>example</Alert>)
    expect(container).toMatchSnapshot()
  })
  it('should render as inline', () => {
    const { container } = render(<Alert block={false}>example</Alert>)
    expect(container).toMatchSnapshot()
  })
})
