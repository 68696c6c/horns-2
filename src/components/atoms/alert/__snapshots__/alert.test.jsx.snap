// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Alert should render as block 1`] = `
.emotion-0 {
  color: rgb(23,162,184);
  border-color: rgb(23,162,184);
  font-family: Helvetica,sans-serif;
  font-size: medium;
  font-style: normal;
  font-weight: 400;
  line-height: 1.6em;
  border-top-width: 2px;
  border-top-style: solid;
  border-bottom-width: 2px;
  border-bottom-style: solid;
  border-left-width: 2px;
  border-left-style: solid;
  border-right-width: 2px;
  border-right-style: solid;
  padding-top: 12px;
  padding-bottom: 12px;
  padding-left: 16px;
  padding-right: 16px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.emotion-0 .icon {
  margin-right: 1em;
}

<div>
  <div
    class="alert emotion-0"
    color="info"
  >
    example
  </div>
</div>
`;

exports[`Alert should render as default 1`] = `
.emotion-0 {
  color: rgb(23,162,184);
  border-color: rgb(23,162,184);
  font-family: Helvetica,sans-serif;
  font-size: medium;
  font-style: normal;
  font-weight: 400;
  line-height: 1.6em;
  border-top-width: 2px;
  border-top-style: solid;
  border-bottom-width: 2px;
  border-bottom-style: solid;
  border-left-width: 2px;
  border-left-style: solid;
  border-right-width: 2px;
  border-right-style: solid;
  padding-top: 12px;
  padding-bottom: 12px;
  padding-left: 16px;
  padding-right: 16px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 1.6em;
  min-width: 1.6em;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  vertical-align: middle;
}

.emotion-0 .icon {
  margin-right: 1em;
}

<div>
  <span
    class="alert emotion-0"
    color="info"
  >
    example
  </span>
</div>
`;

exports[`Alert should render as inline 1`] = `
.emotion-0 {
  color: rgb(23,162,184);
  border-color: rgb(23,162,184);
  font-family: Helvetica,sans-serif;
  font-size: medium;
  font-style: normal;
  font-weight: 400;
  line-height: 1.6em;
  border-top-width: 2px;
  border-top-style: solid;
  border-bottom-width: 2px;
  border-bottom-style: solid;
  border-left-width: 2px;
  border-left-style: solid;
  border-right-width: 2px;
  border-right-style: solid;
  padding-top: 12px;
  padding-bottom: 12px;
  padding-left: 16px;
  padding-right: 16px;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  height: 1.6em;
  min-width: 1.6em;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  vertical-align: middle;
}

.emotion-0 .icon {
  margin-right: 1em;
}

<div>
  <span
    class="alert emotion-0"
    color="info"
  >
    example
  </span>
</div>
`;
